### ISSUE TITLE
Grouping and deriving from a template class leads to "not documented" members.

### ISSUE THREAD

**Issue by michaelcadilhac** [doxygen/doxygen#11095]
**Describe the bug**
When using both grouping and templates, Doxygen complains that all but the first element of the group do not have documentation.  Documentation is however correctly created (inherited). 

**To Reproduce**

Since it is an extremely short test case, I'll inline it:

```c++
/** SA. */
template <typename T>
struct A
{
  public:
    ///@{
    /** Same documentation for all members. Details */
    void f1 ();
    void f2 ();
    void f3 ();
    ///@}
};

/** SB. */
class B : public A<int> {};
```

`Doxyfile`:
```
JAVADOC_AUTOBRIEF  = YES
DISTRIBUTE_GROUP_DOC = YES
```

Running this, doxygen spits out:
```
file.cpp:9: warning: Member f2() (function) of class A< int > is not documented.
file.cpp:10: warning: Member f3() (function) of class A< int > is not documented.
```

**Expected behavior**
`f2` and `f3` _are_ documented, this warning should not appear.
 
**Version**
1.12.0 on Arch Linux latest from official sources.

**Comment by albert-github** [comment]
It looks like that for templated classes the function `hasUserDocumentation()` as used in `void MemberDefImpl::warnIfUndocumented() const` does not know that there is brief and detailed documentation available, better said  there is no `DeepCopyUniquePtr<DocInfo>` available for `f2` and `f3` in case of a templated class.
In case of a "normal" class all looks like to be OK.

Extended example: [example.tar.gz](https://github.com/user-attachments/files/16728313/example.tar.gz)

**Comment by doxygen** [comment]
@michaelcadilhac Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
